src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusEntityType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    /// <summary>
+    /// Service Bus entity type.
+    /// </summary>
+    public enum ServiceBusEntityType
```

Comment:
Let's just name this EntityType - it's in the ServiceBus namespace so it is scoped.